app/views/user_mailer/add_email.html.erb

```diff
@@ -13,11 +13,7 @@
               <td>
                 <center>
                   <%= link_to t('user_mailer.email_confirmation_instructions.link_text'),
-                              add_email_confirmation_url(
-                                _request_id: @request_id,
```

Comment:
As far as I can tell, `@request_id` was never assigned... so it's always been `nil` in this email (it does get used in `email_confirmation_instructions` but that's a separate one)